pipelines/src/pipelines/tensorflow/prediction/pipeline.py

```diff
@@ -88,23 +89,20 @@ def tensorflow_pipeline(
     queries_folder = pathlib.Path(__file__).parent / "queries"
 
     ingest_query = generate_query(
```

Comment:
maybe rename `ingest_query` and the `ingest` task to `preprocessing_query` and `preprocessing`

Reply:
done